Tests/SodiumTests/SodiumTests.swift

```diff
@@ -270,6 +272,18 @@ class SodiumTests: XCTestCase {
         XCTAssertFalse(sodium.pwHash.strNeedsRehash(hash: hash!, opsLimit: sodium.pwHash.OpsLimitInteractive, memLimit: sodium.pwHash.MemLimitInteractive))
         XCTAssertTrue(sodium.pwHash.strNeedsRehash(hash: hash!, opsLimit: sodium.pwHash.OpsLimitSensitive, memLimit: sodium.pwHash.MemLimitSensitive))
     }
+    
+    func testSHA256Hash() {
+        let message = "My Test Message".bytes
+        let h = sodium.utils.bin2hex(sodium.sha256Hash.hash(message: message)!)!
+        XCTAssertEqual(h, "06b87ce47e9cb96156a4f55e405c2587fff2be2ed11adb9f465dc964ef70938e")
+    }
+    
+    func testSHA512Hash() {
+        let message = "My Test Message".bytes
+        let h = sodium.utils.bin2hex(sodium.sha512Hash.hash(message: message)!)!
+        XCTAssertEqual(h, "4173c1218c42b37943f39a7c5a87b782ae945b8609b8dae66a0552c56abd931ba0561a6df052008c1994a505024c31f183a5c34c0dda1e0ce82e9b9e7ea08459")
+    }
```

Comment:
I would align the tests with libsodium for consistency and to ensure that the values are identical across language boundaries, see [`libsodium/test/default/hash.c`](https://github.com/jedisct1/libsodium/blob/59b8ba3573b6cde4c2dfca96326175c30b07961b/test/default/hash.c).

Reply:
I'm not finding any test calling `crypto_hash_sha256` and asserting on the resulting hash in that file or in libsodium in general, only resulting bytes getting printed and assertions about hash digest sizes. Am I missing something? Or would you like me to grab the printed output from hashing `static unsigned char x[] = "testing\n";` and assert against it here? All the other tests in `SodiumTests.swift` use `let message = "My Test Message".bytes` so this seemed like the most consistent way to write the new tests.

Reply:
I added the test case from libsodium to the sha256 test. There was none that I could find for sha512.

Reply:
`crypo_hash()` is  called in the linked test, which defaults to `crypto_hash_sha512()`, i.e., the first two lines of the `hash.exp` file contain the expected SHA-512 hashes for the values of `x` and `x2` respectively.

Reply:
Ah I see. Added in latest.